### ISSUE TITLE
Rendering Issue with Bloodmoon

### ISSUE THREAD

**Issue by SpaceRiceBowl** [Tschipp/CarryOn#24]
Discovered a rather small bug when playing around with this mod in creative.
![2017-08-27_19 53 11](https://user-images.githubusercontent.com/29825468/29755176-d3a2bb2a-8b61-11e7-8309-01517c63b12b.png)

It seems that the red tint doesn't apply to the carried chest in first person. Works in third person perfectly fine though.

**Comment by Tschipp** [comment]
Yes, the render in first person also doesn't reflect lighting changes. We are working on fixing that.
